### PR TITLE
Add documentation for 'accessor' function

### PR DESCRIPTION
**Pull request by patoarvizu**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | documents [bank-vaults #1651](https://github.com/banzaicloud/bank-vaults/pull/1651)
| License         | Apache 2.0

### What's in this PR?
Documents the `accessor` function added in [bank-vaults #1651](https://github.com/banzaicloud/bank-vaults/pull/1651).

### Why?
The function added needs documentation.

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] User guide and development docs updated (if needed)
- [x] Related Helm chart(s) updated (if needed)

### CODE REVIEW

**Review by fekete-robert — APPROVED** [review]

**Comment by patoarvizu** [comment]
@fekete-robert Do we now just wait until the corresponding PR in bank-vaults gets merged before merging this?

**Comment by fekete-robert** [comment]
@patoarvizu Yes, it would be best if we merged the docs after the related PR has been released in bank-vaults

**Review by Rasek91 — APPROVED** [review]

**Pull request merged**
